docs/pages/workflow/glossary-of-terms.md

```diff
@@ -111,7 +111,7 @@ The `eas.json` file used to configure [EAS CLI](#eas-cli). For more information,
 
 ### EAS Metadata
 
-A command line tool for uploading and downloading Apple App Store metadata as JSON. This tool is available in the `eas-cli` package and should be used to improve the iOS submission process.
+A command line tool for uploading and downloading Apple App Store metadata as JSON. This tool is available in the [EAS CLI](#eas-cli) package and should be used to improve the iOS submission process, see [EAS Metadata](../eas/metadata/index.md).
```

Comment:
```suggestion
A command line tool for uploading and downloading Apple App Store metadata as JSON. This tool is available in the [EAS CLI](#eas-cli) package and should be used to improve the iOS submission process, see [EAS Metadata](/eas/metadata/index).
```

Reply:
I'm not sure if I follow the suggestion, by using file references we get:
- vscode autocompletion
- validatable links and headers

for example: 
![image](https://user-images.githubusercontent.com/1203991/189142339-68bc6cfe-6d23-4407-9bd7-772a47112338.png)

Reply:
Relative paths are pain when moving the content around, so we stick to the full path.

Removing `.md` is future-proofing, this will bite use when we rename all the files to `.mdx`.

Can you explain a part about links validation? I think it works anyway (if you have the Next validation in mind), but maybe you are talking about remark script?

Reply:
@byCedric I find it's easy to do this by opening the `docs/pages` folder directly in vs code, then all the auto-complete works as expected.

Reply:
Next has a lot of configurable options, defined in `next.config.js`. That's on top of the page routing, all of which Markdown itself is unaware of.

This makes validating links pretty hard without Next. Yes, we can spin up the next server, but that would be slow compared to resolving based on the filepath. (and ofc, you get a better workflow in vscode)

Reply:
Still not sure about what exactly validation process of link you are talking about, can you link me a file?

The problem is that even with `.md` and relative paths we still ship broken links to PROD, so I'm not sure if this validation process is working right now, see:
* https://github.com/expo/expo/pull/18895

Additionally, we have committed to correcting those links a good while ago, and currently we are at place where our content is vastly mixed when it comes to the links, also all new content crated do not include the extension in links, so adding `.md` back to the links might be a quite large task.

Maybe we should find or develop new links validation method, which is agnosting when it comes to the form in which links are defined in MD files?

Reply:
> The problem is that even with .md and relative paths we still ship broken links to PROD, so I'm not sure if this validation process is working right now, see:
It's the [`lint-links` command](https://github.com/expo/expo/blob/main/docs/package.json#L17). Because we can statically evaluate these links, we can validate them.

You are right that we still ship broken links to production, I turned it off because we have a lot of issues already + it takes a long time to run it in CI.
![image](https://user-images.githubusercontent.com/1203991/189185937-7a9fbe36-e638-46fd-892f-dbc2466dc49f.png)

I'm open to other solutions though, I think we just lose some benefits if we go for url-based linking.